settings.js

```diff
@@ -133,6 +149,9 @@ function getNewWorkspaceSettings(index) {
     list.push(uuid);
     workspaceList.set_strv('list', list);
     settings.set_int('index', index);
+    settings.set_string('color', getUnusedColor());
+    let n = findWorkspaceSettingsByName(/^Unnamed .*/).length;
+    settings.set_string(`Unnamed ${n+1}`)
```

Comment:
Is there anything which prevents us from using `Workspace ${i}` here?

Reply:
No, as long as the list is "monotonic" (which it is atm.) that would also work

Reply:
Right, I'd vote for `Workspace n` in that case, as that makes it clearer that it's a workspace related.